### ISSUE TITLE
Base Chapter file with Hero Image lacks Alt text; has empty attribute

### ISSUE THREAD

**Issue by airbr** [HTTPArchive/almanac.httparchive.org#3889]
Incredibly good work on this project. I feel like the banner image that is a prominent hero image and should have some alt text, it has an attribute but no value currently.

![Image](https://github.com/user-attachments/assets/4738cfea-4fb7-4da6-8074-cc933ae9b357)

This would have palpable benefits for the share-ability of the website, provide Alt text for accessibility and SEO. At least, that is my rough understanding.

**Comment by airbr** [comment]
I guess if I had to suggest a benefit,  search engines will come up with results based on alt text, and could increase the visibility to some degree? This is just my work-influenced brain thinking.

**Comment by airbr** [comment]
Oh second thought - the accessibility chapter itself says that empty alt tags are ok for decorative images. This banner is arguably decorative. But would this harm?

"Mascots for web sustainability work on renewable energy creation"?

**Comment by tunetheweb** [comment]
Yes I think this was intentional as it’s decorative.

I’m not sure I agree with that though since it does set the tone for the chapter.
